internal/cloud/backend.go

```diff
@@ -437,6 +439,51 @@ func (b *Cloud) retryLogHook(attemptNum int, resp *http.Response) {
 	}
 }
 
+func (b *Cloud) Workspace(name string) (*tfe.Workspace, error) {
+	workspace, err := b.client.Workspaces.Read(context.Background(), b.organization, name)
+	if err != nil {
+		return nil, err
+	}
+	return workspace, nil
+}
```

Comment:
From a style perspective, I'm not too crazy about this helper. I think the function body is the equivalent of `return b.client.Workspaces.Read(context.Background(), b.organization, name)` so wrapping it doesn't really add value.

Reply:
Good catch. I've updated this.